tests/auth.test.ts

```diff
@@ -3,11 +3,11 @@ import { configureApp } from '../src/server';
 import { describe, expect, it, jest ,beforeAll,afterAll} from '@jest/globals';
 import AuthService from '../src/service/auth.service';
 import {server} from "../src/server"
+import jwt from 'jsonwebtoken'
+import User from '../src/models/user.model';
 jest.mock('../src/service/auth.service');
 
 const app = configureApp() ; // Replace with your app entry point
```

Comment:
You may remove this line if not being used,

Reply:
@Oliviier-dev